shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/Apple.kt

```diff
@@ -56,7 +56,8 @@ class AppleConfigurablesImpl(
         properties.getProperty("useFixedXcodeVersion")?.let { requiredXcodeVersion ->
             val currentXcodeVersion = xcode.version
 
-            if (currentXcodeVersion != requiredXcodeVersion) {
+            if (properties.getProperty("ignoreXcodeVersionCheck") != "true" &&
+                    currentXcodeVersion != requiredXcodeVersion) {
```

Comment:
Does it make some sense to add hint about `ignoreXcodeVersionCheck` to the error message below?

Reply:
Done